include/inja/parser.hpp

```diff
@@ -655,11 +655,11 @@ class Parser {
         }
       } break;
       default: {
+        current_block = nullptr;
```

Comment:
This change isn't correct, the parser should keep current_block pointing to the token where an error arises.